### DISCUSSION TITLE
New Module for DiscordJs

### DISCUSSION THREAD

**Discussion by benjGam** [discordjs/discord.js#9917] [Ideas]
Hi, i would like to ask if i wish to create a new huge discordjs Node Package (as apitypes package i guess). So my question is : Is that okay to prefix my package as @discordjs or it's not recommended to do so ?

**Comment by Jiralite** [comment]
Sounds like you're talking about https://github.com/discordjs/discord-api-types?

**Reply by benjGam** [reply]
Not exactly, i mean, i want to create my own package and then publish it on NPM Registry, so i would like to know if it's okay to prefix the package name as '@discordjs' or if i can't do that, and you guys prefer me to post without prefix.

**Reply by benjGam** [reply]
And yes, i talked about that, but in fact : I dunno who's managing "api-types" repository

**Reply by vladfrangu** [reply]
> And yes, i talked about that, but in fact : I dunno who's managing "api-types" repository

I am, why? 👀

**Comment by almostSouji** [comment]
> Is that okay to prefix my package as `@discordjs` or it's not recommended to do so?

> When you sign up for an npm user account or create an organization, you are granted a scope that matches your user or organization name. You can use this scope as a namespace for related packages. [src](https://docs.npmjs.com/about-scopes)
> 
> A scope allows you to create a package with the same name as a package created by another user or organization without conflict.

TL;DR: no
